test/_whitelist.js

```diff
@@ -0,0 +1,438 @@
+	'https://github.com/hobbyquaker/awesome-mqtt',
+	'https://github.com/diasdavid/awesome-hacking-spots',
+	'https://github.com/cristianoliveira/awesome4girls',
+	'https://github.com/vorpaljs/awesome-vorpal',
+	'https://github.com/vinjn/awesome-vulkan',
+	'https://github.com/egeerardyn/awesome-LaTeX',
+	'https://github.com/antontarasenko/awesome-economics',
+	'https://github.com/sublimino/awesome-funny-markov',
+	'https://github.com/danielecook/Awesome-Bioinformatics',
+	'https://github.com/Siddharth11/Colorful',
+	'https://github.com/scholtzm/awesome-steam',
+	'https://github.com/hackerkid/bots',
+	'https://github.com/dastergon/awesome-sre',
+	'https://github.com/KimberlyMunoz/empathy-in-engineering',
+	'https://github.com/xen0l/awesome-dtrace',
+	'https://github.com/brunocvcunha/awesome-userscripts',
+	'https://github.com/tobiasbueschel/awesome-pokemon',
+	'https://github.com/exAspArk/awesome-chatops',
+	'https://github.com/kdeldycke/awesome-falsehood',
+	'https://github.com/heynickc/awesome-ddd',
+	'https://github.com/woop/awesome-quantified-self',
+	'https://github.com/hbokh/awesome-saltstack',
+	'https://github.com/nicolesaidy/awesome-web-design',
+	'https://github.com/aliesbelik/awesome-jmeter',
+	'https://github.com/terkelg/awesome-creative-coding',
+	'https://github.com/aviaryan/awesome-no-login-web-apps',
+	'https://github.com/TheJambo/awesome-testing',
+	'https://github.com/johnjago/awesome-free-software',
+	'https://github.com/podo/awesome-framer',
+	'https://github.com/BubuAnabelas/awesome-markdown',
+	'https://github.com/mislavcimpersak/awesome-dev-fun',
+	'https://github.com/awkward/awesome-netherlands-events',
+	'https://github.com/kakoni/awesome-healthcare',
+	'https://github.com/DavidLambauer/awesome-magento2',
+	'https://github.com/xiaohanyu/awesome-tikz',
+	'https://github.com/analyticalmonk/awesome-neuroscience',
+	'https://github.com/johnjago/awesome-ad-free',
+	'https://github.com/angrykoala/awesome-esolangs',
+	'https://github.com/roaldnefs/awesome-prometheus',
+	'https://github.com/hobbyquaker/awesome-homematic',
+	'https://github.com/sfischer13/awesome-ledger',
+	'https://github.com/johnjago/awesome-uncopyright',
+	'https://github.com/kennethreitz/awesome-coins',
+	'https://github.com/folkswhocode/awesome-diversity',
+	'https://github.com/zachflower/awesome-open-source-supporters',
+	'https://github.com/robinstickel/awesome-design-principles',
+	'https://github.com/mojoaxel/awesome-regression-testing',
+	'https://github.com/johnjago/awesome-theravada',
+	'https://github.com/inspectit-labs/awesome-inspectit',
+	'https://github.com/nayafia/awesome-maintainers',
+	'https://github.com/xxczaki/awesome-calculators',
+	'https://github.com/ZYSzys/awesome-captcha',
+	'https://github.com/markusschanta/awesome-jupyter',
+	'https://github.com/andrewda/awesome-frc',
+	'https://github.com/engagingspaces/awesome-humane-tech',
+	'https://creativecommons.org/publicdomain/zero/1.0/',
+	'https://sindresorhus.com/'
```

Comment:
The last two links can be removed from the whitelist since they wouldn't be linted anyways.

Reply:
Good call 😄